jetty-core/jetty-security/src/main/java/org/eclipse/jetty/security/SecurityHandler.java

```diff
@@ -46,28 +49,32 @@
  * Select and apply an {@link Authenticator} to a request.
  * <p>
  * The Authenticator may either be directly set on the handler
- * or will be create during {@link #start()} with a call to
+ * or it will be created during {@link #start()} with a call to
  * either the default or set AuthenticatorFactory.
  * <p>
- * SecurityHandler has a set of initparameters that are used by the
+ * SecurityHandler has a set of parameters that are used by the
  * Authentication.Configuration. At startup, any context init parameters
  * that start with "org.eclipse.jetty.security." that do not have
  * values in the SecurityHandler init parameters, are copied.
+ *
+ * TODO rename to ConstraintHandler
  */
-public abstract class SecurityHandler extends Handler.Wrapper implements Authenticator.AuthConfiguration
+public abstract class SecurityHandler extends Handler.Wrapper implements AuthConfiguration
```

Comment:
yeah... I'm on the fence.  It is a better name, but it will change a lot of legacy examples, doco etc.
Let's fix all the other issues and ponder this one.